docs/new-feature-tutorial.md

```diff
@@ -11,12 +11,44 @@ real-time push system; the
 [real-time push and events](events-system.html) documentation has a
 detailed explanation of how everything works.
 
-## General Process in brief
+## General Process
+
+### Files impacted
+
+This tutorial will walk through adding a new feature to a Realm (an
+organization in Zulip). The following files are involved in the process:
+
+**Backend**
+- `zerver/model.py`: Defines the database model.
+- `zerver/lib/actions.py`: Contains code for updating and interacting with the database.
+- `zerver/lib/events.py`: Involved in updating the state of the application after
+  an event (for example, an administrator changing the value of a realm setting).
```

Comment:
Maybe "Ensures that the state Zulip sends to clients is always consistent and correct"